packs/autoscale/actions/workflows/asg_delete_node.yaml

```diff
@@ -87,8 +79,6 @@ workflows:
           - failure_get_dns_zone_id
       get_dns_record_id:
         action: rackspace.find_dns_record_id
-        policies:
-          wait-before: 1
         input:
           zone_id: <% $.dns_zone_id %>
```

Comment:
Good to know. I debated this pattern though with Action-Chain: the explicit publish allows me to avoid the tight coupling of a variable with a task... letting me move around how I retrieve the value in the context of the workflow. So, good to know we have, but probably something I'd avoid as a general rule.